lib/vec.dl

```diff
@@ -147,6 +147,16 @@ function fold(v: Vec<'A>, f: function('B, 'A): 'B, initializer: 'B): 'B {
     res
 }
 
+// Gets the first element of a vector if it exists
+function first(vec: Vec<'T>) -> Option<'T> {
```

Comment:
There's actually a syntax error here and in the other new function: `->` should be `:`.